### PR TITLE
CSS Update — fix some darkmode elements

### PR DESCRIPTION
**Pull request by WallyTheBully**
Teambuilder files manager (darkmode): https://prnt.sc/fjtkgm
Menugroup (darkmode): https://prnt.sc/fjtt1g
Pm Box (darkmode / darkmode notification): https://prnt.sc/fjued6

### CODE REVIEW

**Comment by xfix** [comment]
Looks great.

**Comment by Zarel** [comment]
The 3D was somewhat important for the dropdowns though :s

**Comment by WallyTheBully** [comment]
True @Zarel , updating it asap (box shadow and avoiding transparent background considering the 3d)

**Comment by Zarel** [comment]
I mean, you could have a translucent 3D effect, it would look glassy which would be kinda cool.

**Comment by WallyTheBully** [comment]
updating it rn, I'll send sceenshots, as you said drag & drop 3d effect is really important on the teambuilder and I was about to modify it when you commented, anyways, I agree with you

**Comment by WallyTheBully** [comment]
https://prnt.sc/fjvi3c

would it be better like this? (glassy effect; 3d looks alright in my opinion)

**Comment by Zarel** [comment]
It's okay, it looks a bit similar to other buttons but I guess that's not a huge problem.

File: style/client.css [hunk 1]
```diff
@@ -117,11 +117,10 @@ button:disabled {
...
 .dark .textbox {
-	border-color: #888888;
-
-	box-shadow: none;
-	background: #282B2D;
-	color: #DDD;
+    border-color: #888888;
+    box-shadow: none;
+    background: rgb(60, 59, 58);
+    color: #DDD;
```
Comment (Zarel) [inline style/client.css]: Tabs.........

**Comment by WallyTheBully** [comment]
Done @Zarel waiting for approval, and yeah it looks a bit similar but I might be able to improve it, if so, I'll make another pull request

**Comment by Zarel** [comment]
Ummm.... this is not ok

image

No line should start with a space.

**Comment by Zarel** [comment]
It's still not fixed.

**Comment by WallyTheBully** [comment]
sorry, fixed it, it should be fine now (removed the inappropriate space and added a tab)

**Comment by panpawn** [comment]
Nope, those are still spaces

File: style/client.css [hunk 2]
```diff
@@ -117,26 +117,29 @@ button:disabled {
 }
 
 .dark .textbox {
-	border-color: #888888;
-
-	box-shadow: none;
-	background: #282B2D;
-	color: #DDD;
+    	border-color: #888888;
```
Comment (Zarel) [inline style/client.css]: This has four extraneous spaces.

File: style/client.css [hunk 3]
```diff
@@ -117,26 +117,29 @@ button:disabled {
...
+    	box-shadow: none;
```
Comment (Zarel) [inline style/client.css]: This too

File: style/client.css [hunk 4]
```diff
@@ -117,26 +117,29 @@ button:disabled {
...
+    	background: rgb(60, 59, 58);
```
Comment (Zarel) [inline style/client.css]: This too

File: style/client.css [hunk 5]
```diff
@@ -117,26 +117,29 @@ button:disabled {
...
+    	color: #DDD;
```
Comment (Zarel) [inline style/client.css]: This too

File: style/client.css [hunk 6]
```diff
@@ -3340,22 +3345,56 @@ a.ilink.yours {
 /* teambuilder */
 
 .dark .folderpane {
-	border-left-color: #77838c;
+	border-left-color: #484848;
 }
 
+.dark .folderpane {
+    	border-left-color: #484848;
```
Comment (Zarel) [inline style/client.css]: This too

**Comment by panpawn** [comment]
@Zarel, I think they all do

File: style/client.css [hunk 7]
```diff
@@ -3340,22 +3345,56 @@ a.ilink.yours {
...
 .dark .folderpane h3,
 .dark .folder .selectFolder,
 .dark .folderhack1,
 .dark .folderhack2 {
-	background: #77838c;
+	background: #484848;
+}
+
+.dark .folderpane h3 {
+    	margin: 0;
```
Comment (Zarel) [inline style/client.css]: This too, etc etc for a lot of the following lines

**Comment by Zarel** [comment]
No, one of them doesn't!

**Comment by Zarel** [comment]
(The one I skipped)

**Comment by WallyTheBully** [comment]
could it be fixed, by the way, tell me and I'll make a request asap


fixed;

**Comment by panpawn** [comment]
That's a lot of contrast

**Comment by WallyTheBully** [comment]
I agree with you @panpawn, the white variant looks better, @Zarel what's your opinion on that, do you think that the left and right bar should be black when using darkmode?

**Comment by Zarel** [comment]
The battle contents should not change at all between dark and light modes.

**Comment by WallyTheBully** [comment]
11css

a'ight, what about it?

**Comment by WallyTheBully** [comment]
11css
preview with another background

**Comment by Zarel** [comment]
Are those broken in all themes? Just in dark mode because something interferes with it?

**Comment by WallyTheBully** [comment]
those (reflect & light screen, not weathers in general)are broken in dark mode only beacause I assume, something interferes with it

**Comment by Zarel** [comment]
Can you investigate?

**Comment by WallyTheBully** [comment]
yeah apparently, the "weather" class isn't a thing in darkmode 

`.weather {
    position: absolute;
    display: block;
    top: 0;
    left: 0;
    right: 0;
    bottom: 0;
    text-shadow: #FFFFFF 1px 1px 0, #FFFFFF 1px -1px 0, #FFFFFF -1px 1px 0, #FFFFFF -1px -1px 0;
}`

I'd suggest to add the following code to the client CSS, allowing it to be readable 

`.dark .weather {
color: #000;
}`

this is how the white mode displays those screens moves ;



only "light screen" and "reflect" are affected by this "bug" and it should be like this in darkmode since you said ;

> The battle contents should not change at all between dark and light modes.

**Comment by Zarel** [comment]
You're going to need to be clearer than "isn't a thing in darkmode"

**Comment by WallyTheBully** [comment]
yes, the weather CSS class wasn't coded, as a consequence it isn't on the client CSS file, I'd suggest to add it, sorry for the lack of precision in my last comment

**Comment by Zarel** [comment]
It doesn't need to be, the `.weather` CSS class should cover it.

**Comment by WallyTheBully** [comment]
this is true, then we should just add `color: #000;` to the `.weather` class

**Comment by Zarel** [comment]
yes

**Comment by Zarel** [comment]
I believe this has mostly been superseded by other changes, and can be closed now?

**Comment by Zarel** [comment]
Tested, this is fixed in the current version of PS.

**Pull request closed without merging**